benchmarks/decoders/benchmark_decoders_library.py

```diff
@@ -534,9 +526,13 @@ def run_benchmarks(
                 df_item["median"] = results[-1].median
                 df_item["iqr"] = results[-1].iqr
                 df_item["type"] = f"{kind}:seek()+next()"
-                df_item["fps"] = 1.0 * num_samples / results[-1].median
-                df_item["fps_p75"] = 1.0 * num_samples / results[-1]._p75
-                df_item["fps_p25"] = 1.0 * num_samples / results[-1]._p25
+                df_item["fps_mean"] = num_samples / results[-1].mean
```

Comment:
This is not the arithmetic mean fps. Because it's the arithmetic mean of 1/fps -- which means it's really the harmonic mean

Reply:
Yes, agreed. That's due to us reporting fps instead of time. Do you think this is a problem?

Reply:
Wait, I'm not sure I understand @ahmadsharif1 's comment. Are we reporting `fps` or `1 / fps`?

Reply:
Actually, I don't think this is the harmonic mean, but I think we do want the harmonic mean. In order to get it, I think we need to do:
```
df_item["fps_mean"] = num_samples / np.sum([1/t for t in results[-1].times])
```